### PR TITLE
✨  Splashpage

### PR DESCRIPTION
**Pull request by SilviaBarcellini**
The Splash Page is the Welcome Page of the App. It is the first page the user sees every time the app is opened.
- add splash page
- add link to index page
- add logo
- update css globals
- style page container

### CODE REVIEW

**Review by mhstrkmp — COMMENTED** [review]
lgtm 🎉

**Review by RonjaMeinecke — APPROVED** [review]
🚀GJ

File: pages/splash.tsx [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+import Logo from "../components/logo/logo";
+
+export default function SplashPage() {
+  return (
+    <div>
+      <div className={styles.logo}>
+        <Logo logo="bakend" size="big" />
+      </div>
+      <div>
+        <a className={styles.link} href="http://localhost:3000">
```
Comment (DNaunin) [inline pages/splash.tsx:11]:
```suggestion
        <a className={styles.link} href="/">
```

To link to Home globally you can just "/".

**Review by DNaunin — COMMENTED** [review]
Good Job!  🚀

File: components/openNewPage/openNewPage.tsx [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import styles from "../openNewPage/openNewPage.module.css";
+
+export default function OpenNewPage() {
+  return (
+    <a href="http://localhost:3000/splash">
```
[the remark this answers is not in the document]
Comment (SilviaBarcellini) [reply components/openNewPage/openNewPage.tsx:5]: Will do, thank you so much!!

File: pages/splash.tsx (hunk 1 above)
Comment (SilviaBarcellini) [reply pages/splash.tsx:11]: Thank you!!

**Pull request merged**